brabbuss/moodytunes
# Feature failures

#### What does  this PR do?
- Reformats & adds stylings to the Form options.
- Adds an 'error-handling' page for if there are no song results returned based off of a user's chosen options.
- Clears past results within the ResultsView. 

#### How should this be manually tested?

- Hover over any of the options provided on the Form to see border-styling over that specific option.
- Click on any of the options within the Forms categories to see the styling effects.

#### Any background context you want to provide?
Currently, after selecting an option for each category & clicking the 'Get songs' button, the error-handling page briefly appears before displaying the actual results.
